documentation.md

```diff
@@ -0,0 +1,14 @@
+# Documentation
+This guide aims to make it clear what information can and cannot be included in documents and designs that are made public.
+
+## Restricted Items
+1. Network ranges and IP addresses <br>
+The organisation currently have these classed as _sensitive_
+1. S3 Bucket names <br>
```

Comment:
We will also care about things such as the (non-secret) elements of access keys. Might be worth the two of us having a skim-review through our code base to pull out all of the other examples of things like this, to distill a minimalist set of principles out.